antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+set -e
+
+BASEDIR=$(dirname "$(readlink -f "$0")")
+
+CURRENT_TAG=${git describe --tag}
+
+createReleaseDocumentation() {
+  "${BASEDIR}"/execute_github_site_maven_plugin.sh
+}
+
+deleteOldSnapshotFolderInGhPages() {
+  fetchGhPages
+  git checkout gh-pages
+  OLD_SNAPSHOT_DIR=$(ls -d *SNAPSHOT*/)
+  rm -rf "*SNAPSHOT*"
```

Comment:
why not use `$OLD_SNAPSHOT_DIR` here?

Reply:
The `-f` flag should not be necessary

Reply:
```suggestion
  rm -rf "*SNAPSHOT"
```

Reply:
I would actually instead prefer a call to find, instead of globbing:
```suggestion
  find . -iname "*SNAPSHOT" -type d -maxdepth 1 \
      -exec rm -r {} \; \
      -exec git commit -am "Remove {} documentation folder" \;
```